clientv3/watch.go

```diff
@@ -52,6 +59,22 @@ type WatchResponse struct {
 	Canceled bool
 }
 
+// IsCreateEvent returns true if the event tells that the key is newly created.
+func IsCreateEvent(e *Event) bool {
```

Comment:
That's great comments. I added the tests too.

One concern I have is that we should be careful once we provide previous value in Delete event. For example, if we put it in "Kv" then it would be Modify event too.